storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -867,4 +872,22 @@ public static void setSubscriptionsFilter(SubscriptionsFilter value) {
     public static boolean shouldShowSubscriptionTitle() {
         return prefs.getBoolean(PREF_SUBSCRIPTION_TITLE, false);
     }
+
+    public static void setAllEpisodeSortOrder(SortOrder s) {
+        prefs.edit()
+                .putString(PREF_SORT_ALL_EPISODES, "" + s.code).apply();
+    }
+
+    public static SortOrder getAllEpisodeSortOrder() {
+        return SortOrder.fromCodeString(prefs.getString(PREF_SORT_ALL_EPISODES,
```

Comment:
When storing the setting in a new place, we should probably migrate the value to the new place. Otherwise the sort order that users already selected gets reset. The place to do this would be the class PreferenceUpgrader

Reply:
This is done